src/main/java/io/github/programminglife2016/pl1_2016/database/FetchDatabase.java

```diff
@@ -201,7 +201,7 @@ private JSONArray fetchLinks(int threshold) throws SQLException {
      * @return the collection of specimen
      * @throws SQLException thrown if SQL connection or query is not valid
      */
-//    @SuppressWarnings("checkstyle:methodlength")
```

Comment:
This method is not that large to throw a checkstyle error. I don't think that it is necessary to suppress this.